src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -64,6 +66,39 @@ internal struct NameDeclarationInfo
                 return default;
             }
 
+            private static bool IsTupleElementDeclaration(
+                SyntaxToken token, SemanticModel semanticModel, int position,
+                CancellationToken cancellationToken, out NameDeclarationInfo result)
+            {
+                result = IsFollowingTypeOrComma<TupleElementSyntax>(
+                    token,
+                    semanticModel,
+                    tupleElement => tupleElement.Type,
+                    _ => default(SyntaxTokenList),
+                    _ => ImmutableArray.Create(SymbolKind.Local), cancellationToken);
+
+                return result.Type != null;
+            }
+
+            private static bool IsTupleExpressionDeclaration(
+                SyntaxToken token, SemanticModel semanticModel, int position,
```

Comment:
The position argument seems unused.

Reply:
There are a bunch of methods with about the same signature:

```CS
                if (IsTupleElementDeclaration(token, semanticModel, position, cancellationToken, out var result)
                    || IsParameterDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsTypeParameterDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsVariableDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsIncompleteMemberDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsFieldDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsMethodDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsPropertyDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsPossibleOutVariableDeclaration(token, semanticModel, position, typeInferenceService, cancellationToken, out result)
                    || IsTupleExpressionDeclaration(token, semanticModel, position, cancellationToken, out result)
                    || IsPossibleVariableOrLocalMethodDeclaration(token, semanticModel, position, cancellationToken, out result))
```

I would like to keep the signature as is just for symmetry.

Reply:
Saw that. Fine to leave as-is :-)